model/task_build_variant_names_test.go

```diff
@@ -64,28 +64,11 @@ func TestFindUniqueBuildVariantNamesByTask(t *testing.T) {
 		RevisionOrderNumber: 1,
 	}
 	assert.NoError(t, t3.Insert())
-	b4 := build.Build{
```

Comment:
Removed a test case that was testing to see if this query could handle build variant names changing. 
This is no longer necessary since we only fetch build variant names from the most recent 50 versions as opposed to all time.